src/scenes/AcceleratorRouter/ParticipantProjects/Reports/EditMetricModal.tsx

```diff
@@ -131,6 +131,16 @@ export default function EditMetricModal(props: EditMetricModalProps): JSX.Elemen
             required
           />
         </Grid>
+        <Grid item xs={12}>
+          <Checkbox
+            disabled={false}
+            id={'isPublishable'}
+            name={'isPublishable'}
+            label={strings.PUBLISH_TO_FUNDER_PORTAL}
+            value={record.isPublishable}
+            onChange={(newValue: boolean) => onChange('isPublishable', newValue)}
+          />
+        </Grid>
```

Comment:
Does this need to be hidden for console? Or is it available for both console and participants?

Reply:
I think it's available for both, but will check